### ISSUE TITLE
wrapping method, scaled result

### ISSUE THREAD

**Issue by SabraHashemi** [lukasalexanderweber/stitching#31]
Hi, I used this stitching  for 3 images which are from an area (in different directions for movement)
stitch works but the images seem to have a fixed height, and the result is a compressed height view
what should I do? I think it is related to the wrapping method
![result](https://user-images.githubusercontent.com/17723386/199491444-befaaace-ad27-4065-8cb2-e1314722ab7a.png)

**Comment by lukasalexanderweber** [comment]
the final image is cropped to a fixed height.

Try `stitch * --no-crop` to get the result without cropping.

You can play around with the warping methods with this argument:

`--warper_type x`

with x being one of:

```
spherical
plane
affine
cylindrical
fisheye
stereographic
compressedPlaneA2B1
compressedPlaneA1.5B1
compressedPlanePortraitA2B1
compressedPlanePortraitA1.5B1
paniniA2B1
paniniA1.5B1
paniniPortraitA2B1
paniniPortraitA1.5B1
mercator
transverseMercator
```

**Comment by SabraHashemi** [comment]
I used "crop=false" in my script when I'm creating stitcher object,  but I got :
 OpenCV(4.5.5) D:\a\opencv-python\opencv-python\opencv\modules\core\src\alloc.cpp:73: error: (-4:Insufficient memory) Failed to allocate 7423918080 bytes in function 'cv::OutOfMemoryError'

**Comment by SabraHashemi** [comment]
also with just two of my images (I get memory errors with 4 of my large images), I get this image which still seems to be scaled . how can I solve it? 
I don't want crop and I don't want to be scaled

![Uploading result.png…]()

**Comment by lukasalexanderweber** [comment]
> Failed to allocate 7423918080 bytes

You have not enough RAM to process all files in full resolution. You can set the resolution of the images to be stitched by 

`--final_megapix x`

where x are the megapixel the images are downscaled to. Try e.g. `1` for a start and increase if you're not satisfied with the results resolution

> I get this image which still seems to be scaled . how can I solve it?

This can come from two things

1. The images are not related by a [Homography](https://en.wikipedia.org/wiki/Homography_(computer_vision)) or obtained by a affine device (e.g. a scanner)
2. The warping method causes contortions. Have you tried other warping methods?

You can also share the images so I can have a better look

**Comment by SabraHashemi** [comment]
@lukasalexanderweber  : thank you for your help
these are the images I'm working on:
https://github.com/SabraHashemi/sample-images

**Comment by lukasalexanderweber** [comment]
I also got big problems with your images. Can you tell me how they were taken?

**Comment by SabraHashemi** [comment]
I think by Drone DJI
I got a better result by using fisheye as wrapper
any idea?

**Comment by lukasalexanderweber** [comment]
no, maybe you should go through the [tutorial](https://github.com/lukasalexanderweber/stitching_tutorial) to exmine the detailed effects of different warping methods

**Comment by SabraHashemi** [comment]
thank you for the guide, I will but one more question, please
my final image is good enough 
but it seems **rotated** after changing the wrap to "fisheye", I read about fisheye but I think it related to another thing
![result](https://user-images.githubusercontent.com/17723386/200521742-916d7e6e-b74a-4f03-8361-3fcfbcdabeea.png)
